tests/acceptance-tests/wayland/CMakeLists.txt

```diff
@@ -1,5 +1,4 @@
-pkg_check_modules(WLCS REQUIRED wlcs)
-include_directories(${WLCS_INCLUDE_DIRS})
+pkg_check_modules(WLCS REQUIRED IMPORTED_TARGET wlcs)
```

Comment:
Huh. We don't actually *use* the `PkgConfig::WLCS` anywhere? This seems like it might be a latent bug in our existing CMake?

Reply:
I suppose this is so that CMake fails if WLCS is unavailable. This whole subdir is conditional on `MIR_ENABLE_WLCS_TESTS`, so it will only barf when that's `ON`.

Reply:
acceptance-tests/wayland/test_wlcs_display_server.{cpp,h} both #include WLCS headers, so we should be at least pulling *those* header locations from somewhere!

Reply:
I've pushed a commit fixing this